recipe/meta.yaml

```diff
@@ -65,3 +65,7 @@ about:
 extra:
   recipe-maintainers:
     - scopatz
+    - gonuke
+    - baaaaam
```

Comment:
Yeah, conda-forge can tends to be more precise, so I don't know how it will work out.